config/locales/en.yml

```
@@ -14,19 +14,28 @@ en:
           pending: Pending
           processing: Processing
           completed: Completed
+        priority: Priority
+        priorities:
+          low: Low
+          mid: middle
+          high: high
     errors:
+      blank: &blank
+        blank: can't be blank.
       models:
```

要拿來共用的共用話翻譯的層級放到 models 的下層應已足夠，層級應該不需要像現在這樣放得這麼高。
```suggestion
      models:
        blank: &blank
          blank: can't be blank.
```